libdnf/dnf-sack.cpp

```diff
@@ -736,7 +760,8 @@ load_yum_repo(DnfSack *sack, HyRepo hrepo, GError **error)
         fp_primary = solv_xfopen(primary.c_str(), "r");
         assert(fp_primary);
 
-        g_debug("fetching %s", name);
+        g_debug("%s: loading: %s", __func__, fn_repomd);
+        g_debug("%s: loading: %s", __func__, primary.c_str());
```

Comment:
These two messages will end up being hugely misleading in case the loading of repomd fails, because the second log message already says we're loading primary. It's better to split the loading and check each return code separately below.